### DISCUSSION TITLE
Compatibilidade ES6 para IE8  com polyfills, shim

### DISCUSSION THREAD

**Discussion by dougaraujos** [frontendbr/forum#719] [Perguntas e Respostas]
Galera, estamos iniciando o desenvolvimento de um produto (especificamente integração SCORM) que necessita de compatibilidade com browser antigos como IE8. A intenção era desenvolver em ES6 para viabilizar escalabilidade e organização do produto.

Queria saber se vocês tem algum case e experiência de o quão viável está o uso do [es5-shim](https://github.com/es-shims/es5-shim) e outros polyfills.

**Comment by fabiomirandaa** [comment]
Desenvolver pra um IE tão ultrapassado, infelizmente irá te limitar muito. Não vejo como pode fazer algo que não vá pesar muito na sua aplicação.

**Comment by dougaraujos** [comment]
@fabiomirandaa Sem dúvida. A questão é que atendemos grandes corporações como instituições financeiras intermediárias na qual existe uma gama de restrições internas e integrações que tornam inviável uma migração para outros sistemas e browsers.

Minha dúvida era apenas em relação a estratégia de utilizar os benefícios do ES6 para o IE8 com o uso de polyfills/shims já que estamos modelando um produto que será implementado futuramente em browser modernos. Ou, por experiência própria, alguém sugere desenvolvermos uma única versão em ES3.
